modules/navier_stokes/doc/content/source/materials/INSFVSpeedFunctorMaterial.md

```diff
@@ -0,0 +1,11 @@
+# PINSFVSpeedFunctorMaterial
```

Comment:
Need to `git mv` this

Reply:
that and the tests design doc lol
should be good now